packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -277,7 +277,6 @@ export const DoDontItem = React.forwardRef<HTMLDivElement, DoDontItemProps>(
                         )}
                         <div className="tw-w-full tw-flex tw-items-center">
                             <span
-                                className="a-h3"
```

Comment:
Is this not needed anymore?

Reply:
not needed anymore, as it was just used for `font-family: inherit` what got refactored and can't be the case anymore